server/server.go

```diff
@@ -1,22 +1,28 @@
 package server
 
 import (
-	"github.com/kaspanet/kaspad/util/mstime"
 	"sync/atomic"
 
+	"github.com/kaspanet/kaspad/blockdag"
+	"github.com/kaspanet/kaspad/blockdag/indexers"
+	"github.com/kaspanet/kaspad/txscript"
+	"github.com/kaspanet/kaspad/util"
+	"github.com/kaspanet/kaspad/util/mstime"
+
 	"github.com/kaspanet/kaspad/config"
 	"github.com/kaspanet/kaspad/dagconfig"
 	"github.com/kaspanet/kaspad/mempool"
 	"github.com/kaspanet/kaspad/mining"
-	"github.com/kaspanet/kaspad/server/p2p"
 	"github.com/kaspanet/kaspad/server/rpc"
 	"github.com/kaspanet/kaspad/signal"
 )
 
 // Server is a wrapper for p2p server and rpc server
 type Server struct {
```

Comment:
This object is not a server. It encompasses Kaspad's entire state. Please consider renaming it.

Reply:
We decided that we don't want this object at all. Instead, DAG, mempool, p2pServer, rpcServer, etc. should all be started in Main.